ni/nimi-python
# Add nidigital system tests for pin map methods

- [x] This contribution adheres to [CONTRIBUTING.md](https://github.com/ni/nimi-python/blob/master/CONTRIBUTING.md).

- [x] ~I've updated [CHANGELOG.md](https://github.com/ni/nimi-python/blob/master/CHANGELOG.md) if applicable.~

- [x] I've added tests applicable for this pull request

### What does this Pull Request accomplish?

We have the following Pin Map methods that this change adds or documents coverage for:
* `load_pin_map`
  - Already covered by `test_get_pin_results_pin_information`
* `disable_sites`
  - New test `test_disable_sites`
* `enable_sites`
  - New test `test_enable_sites`
* `is_site_enabled`
  - See `test_enable_sites`
  - See `test_disable_sites`

### List issues fixed by this Pull Request below, if any.

### What testing has been done?

Ran the system tests on a system with nidigital 19.0.1 installed.
